crates/re_log_types/src/data_table.rs

```diff
@@ -737,10 +767,15 @@ impl DataTable {
 
 impl DataTable {
     /// Deserializes an entire table from an arrow payload and schema.
+    ///
+    /// If `compute_cell_sizes` is enabled, this will estimate the size of the arrow data within
+    /// each cell and cache it there.
+    /// See [`DataCell::compute_size_bytes`] for more information.
```

Comment:
Could this be done separately by the caller instead?

```rust
let table = DataTable::deserialize(…)?;
table.compute_sizes();
```

That would let us get rid of this ugly `bool` argument, and make the calling code more explicit and readable